api/utils.py

```diff
@@ -162,3 +168,43 @@ def extract_post_params(
             if value is not None:
                 data[param] = value
     return data
+
+
+def get_object_or_json_error(klass, *args, **kwargs):
+    """
+    This is exact replica of `django.shortcuts.get_object_or_404()` function
+    but instead the raised error is a `json_response_error()`.
+
+    Default message is `Not Found` and default HTTP status is `404`.
+
+    http_status_code: int = Status code. Default 404
+    http_error_message: str = Error message. Default `Not Found`
+
+    IMPORTANT NOTE: `htk.middleware.classes.RaiseResponse` MUST be added to
+                    middlewares in settings to make this work!
```

Comment:
- properly capitalize `middlewares` to whatever the Django variable should be
- rename `settings` to `Django settings`
- Replace the `!` with a `.`

Reply:
Done